packages/test/test-end-to-end-tests/package.json

```diff
@@ -92,6 +92,7 @@
     "@fluidframework/runtime-definitions": "^0.36.0",
     "@fluidframework/runtime-utils": "^0.36.0",
     "@fluidframework/sequence": "^0.36.0",
+    "@fluidframework/server-local-server": "^0.1021.0-0",
```

Comment:
we shouldn't directly reference local server in this package

Reply:
That was introduced from the summaries test before, and just missing the dependency.